client/models/channel.coffee

```diff
@@ -4,3 +4,26 @@ sd = require('sharify').data
 module.exports = class Channel extends Backbone.Model
 
   urlRoot: "#{sd.API_URL}/channels"
+
+  getFeatures: ->
+    switch @get 'type'
+      when 'editorial' then {
+        features: ['header', 'superArticle']
+        sections: ['text', 'artworks', 'images', 'image_set', 'video', 'embed', 'callout', 'toc']
+        associations: ['artworks', 'artists', 'shows', 'fairs', 'partners','auctions']
+      }
+      when 'team' then {
+        features: []
+        sections: ['text', 'artworks', 'images', 'image_set', 'video', 'embed', 'callout']
+        associations: []
+      }
+      when 'support' then {
+        features: []
+        sections: ['text', 'artworks', 'images', 'video', 'callout']
+        associations: ['artworks', 'artists', 'shows', 'fairs', 'partners','auctions']
+      }
+      when 'partner' then {
+        features: []
+        sections: ['text','artworks', 'images','video']
+        associations: []
+      }
```

Comment:
Maybe this is a bad idea, but I'm thinking the `Channel` model in the client can be both channels and partners in order to keep things clean in templates + client-side code... All it needs is the `id` and `type` which I can set at the session level.  thoughts?